config/functions.cfg

```diff
@@ -296,39 +296,39 @@ function build_node {
   #Build the elrond utils
```

Comment:
multiversx-utils? or would it be too complicated?

Reply:
Future change... so as not to confuse people.

Reply:
rename in comment?